TESTS/mbed_drivers/timeout/timeout_tests.h

```diff
@@ -0,0 +1,416 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_TIMEOUT_TESTS_H
+#define MBED_TIMEOUT_TESTS_H
+
+#include "mbed.h"
+#include "unity/unity.h"
+#include "rtos/rtos.h"
```

Comment:
"rtos/rtos.h" is included inside "mbed.h"

Reply:
Fixed.